devfiles/che4z/devfile.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: che4z-
+components:
+  - 
+    type: chePlugin
+    id: broadcommfd/cobol-language-support/latest
+  - 
+    type: chePlugin
+    id: bitlang/cobol/latest
+  - 
+    type: chePlugin
+    id: broadcommfd/explorer-for-endevor/latest
+  - 
+    id: eclipse/che-machine-exec-plugin/7.3.0
+    type: chePlugin
+  - 
+    id: eclipse/che-theia/7.3.0
+    type: cheEditor
+  - 
+    alias: zowe-cli
+    type: kubernetes
+    mountSources: true
+    referenceContent: |
+         kind: List
+         items:
+          -
+           apiVersion: v1
+           kind: Pod
+           metadata:
+            name: zowe
+           spec:
+            volumes:
+             - name: zowe
+               persistentVolumeClaim:
+                 claimName: zowe-claim
+            containers:
+             -
+              image: 'ca-docker-brightside-docker.bintray.io/cli-mainframe-basic-stack:2019-11-19T15-56-35'
```

Comment:
This docker image, on OpenShift, could run into issues around user IDs. See [the arbitrary user id patch](https://github.com/eclipse/che-devfile-registry/tree/master/arbitrary-users-patch) we run for other workspace images.

Reply:
We saw the user ID issue on OpenShift, and will look into arbitrary user ID support. 

Is OpenShift support required at this point in time?

Reply:
It's preferable, in my opinion, but if it doesn't interfere with the functionality you need it's not strictly necessary. It does mean, however, that anything that depends on username or `$HOME` directories will behave weirdly. 

We can also potentially add the image `ca-docker-brightside-docker.bintray.io/cli-mainframe-basic-stack:2019-11-19T15-56-35` to the arbitrary user ID patch set -- I can provide steps if you would like to go down that path (basically you would just need to add a line to [base_images](https://github.com/eclipse/che-devfile-registry/blob/master/arbitrary-users-patch/base_images)).

Reply:
>  It does mean, however, that anything that depends on username or $HOME directories will behave weirdly.

Verified this was the case, the Zowe Explorer extension expects the container `WORKDIR` to be set to `/projects`, and the arbitrary user ID results in `WORKDIR` `/`. This results in a failure to load properly. The rest of the stack software works as expected.

I am pretty sure the arbitrary user ID patch set would create issues, so I tried to make some "quick" changes to our docker images using the patch set as a template. Interestingly, the Zowe Explorer extension fails to load at first, but works properly after a browser refresh. I think we need to spend a little more time digging into the details of our images before we can support the arbitrary user ID, but I have no problem going down that path.